docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module

```diff
@@ -7,19 +7,45 @@
 
 use Drupal\Core\Form\FormStateInterface;
 use \Drupal\Core\Render\Element;
+use Drupal\Component\Utility\Html;
 
 /**
  * Implements hook_preprocess_HOOK().
  */
 function hs_paragraph_types_preprocess_paragraph(&$variables) {
-  if (in_array($variables['paragraph']->bundle(), ['hs_collection', 'hs_priv_collection'])) {
-    /** @var \Drupal\paragraphs\ParagraphInterface $paragraph */
-    $paragraph = $variables['paragraph'];
+  /** @var \Drupal\paragraphs\ParagraphInterface $paragraph */
+  $paragraph = $variables['paragraph'];
+  $bundle = $paragraph->bundle();
+
+  // Exit early if it's not a relevant bundle
+  if (!in_array($bundle, ['hs_collection', 'hs_priv_collection'])) {
+    return NULL;
+  }
+
+  $variables['#attached']['library'][] = 'hs_paragraph_types/hs_collection';
+  $variables['attributes']['class'][] = 'hs-paragraph-style';
+
+  // Logic for private collections
+  if ($bundle == 'hs_priv_collection') {
+    // Logic for private collections
     $row_style = $paragraph->get('field_paragraph_style')->getString();
```

Comment:
Should this be the new field `field_bg_color`?

Reply:
No, we didn't change the private collections. We're still using the old field there.